src/components/molecules/URLCard.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styled from "styled-components";
+
+import Card from "./Card";
+
+const LinkCard = styled("a")`
+	&:hover {
+		text-decoration: none;
+	}
+`;
```

Comment:
I think I'd still like this functionality, it seems weird hovering over the "Learn More" text now with no change to the text.

Reply:
If text-decoration isn't on everything will be underlined, so the title, description and the Learn More which looks quite bad.

![image](https://user-images.githubusercontent.com/60689309/113479223-1136c100-948e-11eb-8a79-34de7787598a.png)

That's also why I added the slightly darker box-shadow on the whole card when hovering.